### ISSUE TITLE
SSD classification loss in Object Detection API is different from the original paper . 

### ISSUE THREAD

**Issue by shamanez** [davidsandberg/facenet#418]
In the paper they have used a softmax cross entropy loss. But here in the model is using **sigmoid cross entropy loss** . 

Why is has changed ? What is the effect ?

**Comment by ugtony** [comment]
I think in this project softmax is used, why do you think sigmoid is used?

**Comment by shamanez** [comment]
Check this configuration file . [This says it is weighted sigmoid](https://github.com/tensorflow/models/blob/master/object_detection/samples/configs/ssd_mobilenet_v1_pets.config#L110)

**Comment by ugtony** [comment]
Where is the configuration file used in this project? Are you sure you raise the issue in the right repository?

**Comment by shamanez** [comment]
OMG! I am extremely sorry  . I  really beg your pardon .  Sorry .
